web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -234,7 +234,7 @@
       {/key}
     </section>
   {:else if description}
-    <p class="break-words whitespace-pre-line w-full text-black dark:text-white text-base">{description}</p>
+    <p class="px-4 break-words whitespace-pre-line w-full text-black dark:text-white text-base">{description}</p>
```

Comment:
This seemed like a regression that I fixed here while testing the asset detail endpoint. cc: @martabal